kubernetes/samples/scripts/domain-lifecycle/helper.sh

```diff
@@ -774,6 +774,9 @@ function validatePythonAvailable {
 
 # try to execute kubernetes cli to see whether cli is available
 function validateKubernetesCliAvailable {
+  if [ ${#} -gt 0 ]; then
+    kubernetesCli=$1
```

Comment:
Consider prepending with 'local' so this function doesn't have the (somewhat surprising) side effect of setting a global.